3-live-update/Tiltfile

```diff
@@ -15,14 +20,15 @@ local_resource(
     resource_deps=['deploy'],
 )
 
-docker_build(
+docker_build_with_restart(
     'hello-tilt',
     'out',
+    entrypoint=['dotnet', 'hello-tilt.dll'],
```

Comment:
hmmm...should we also update the built-in entrypoint parameter to accept an argv array? if so, is there an issue tracking that yet?

Reply:
Oh, it already does: https://github.com/tilt-dev/tilt/blob/ba1de77456a46ff72f5de798aaac092f47f481e7/internal/tiltfile/docker.go#L213 we just haven't updated the docs 😅

Reply:
yay! :elephant: :elephant: :elephant: